website/source/docs/enterprise/runs/variables-and-configuration.html.md

```diff
@@ -75,9 +80,42 @@ For any of the `GITHUB_` attributes, the value of the environment variable will
 be the empty string (`""`) if the resource is not connected to GitHub or if the
 resource was created outside of GitHub (like using `terraform push`).
 
+### Personal Variables
```

Comment:
nitpick: I would change the title of this section to `Personal Environment Variables` to match the outline above